### PR TITLE
HTM detector use NuPIC params

### LINKED ISSUE #231: NAB should be updating and using nupic's getScalarMetricWithTimeOfDayAnomalyParams
NAB PR #206 recently made improvements to its own copy of the model parameters.

Htmengine is relying on  nupic.frameworks.opf.common_models.cluster_params.getScalarMetricWithTimeOfDayAnomalyParams() for its model parameters.  

NAB should be using and updating nupic.frameworks.opf.common_models.cluster_params.getScalarMetricWithTimeOfDayAnomalyParams instead of maintaining its own copy of the parameters. Similar idea to "Switch over to using the Anomaly Likelihood class in NuPIC” (https://github.com/numenta/NAB/pull/184).

Per email from Subutai:

> Subject: Re: HTM Engine questions on mailing list
>
> Vitaly,
>
>Yes, I think that would be better.  We should update the params in common_models and NAB should switch over to using it.
>
> -- Subutai

### PR DESCRIPTION
**Pull request by BoltzmannBrain**
Fixes #231, #234, #235

### CODE REVIEW

**Comment by BoltzmannBrain** [comment]
@subutai please review. I'll followup with new results and scores.

File: nab/detectors/numenta/numenta_detector.py [hunk 1]
```diff
@@ -66,28 +68,12 @@ def handleRecord(self, inputData):
 
 
   def initialize(self):
-    calcRange = abs(self.inputMax - self.inputMin)
-    calcPad = calcRange * .2
+    # Get config params, setting the RDSE resolution
```
Comment (subutai) [inline nab/detectors/numenta/numenta_detector.py]: I'm not sure this strategy matches the NAB requirements. Currently you are not allowed to look ahead and use data values to set your parameters. The only exception is that we assume min/max for each metric are known to the detector (this is a reasonable assumption in most practical scenarios).  Passing in the entire dataset to some black box to set parameters however is not allowed.

File: nab/detectors/numenta/numenta_detector.py [hunk 2]
```diff
@@ -66,28 +68,18 @@ def handleRecord(self, inputData):
...
+    rangePadding = abs(self.inputMax - self.inputMin) * 0.2
```
Comment (BoltzmannBrain) [inline nab/detectors/numenta/numenta_detector.py]: @subutai I tried several different values for the range padding and found 0.2 to give the best results.

**Comment by BoltzmannBrain** [comment]
@subutai scores increased to 65.27, 58.59, and 69.38 (standard, low FP, low FN). Good to merge?

**Comment by subutai** [comment]
:+1: OK to merge (after the nupic merge is complete).  Thanks!

**Pull request merged**